communitiesuk/funding-service-design-docker-runner
# BAU - Debug mode for containers

### Change description
This PR contains README and configuration for running the containers in the docker runner in debug mode, so that we can connect the VS Code debugger to them.



### How to test
Checkout this branch, run docker runner with `docker compose up`, then add launch config as per the README and add breakpoints to check it all works.
